### PR TITLE
Small change to schema syntax

### PR DESCRIPTION
**Pull request by aaronjeline**
Small change to how actions look, removes the `appliesTo` keyword
[Rendered](https://github.com/cedar-policy/rfcs/blob/0024-change/text/0024-schema-syntax.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: text/0024-schema-syntax.md [hunk 1]
```diff
@@ -39,11 +39,15 @@ entity List in [Application] {
...
+action CreateList, GetLists { 
+    principal: [User], 
+    resource: [Application] 
+};
 
-action GetList, UpdateList, DeleteList, CreateTask, UpdateTask, DeleteTask, EditShares
-    appliesTo { principal: [User], resource:[List] };
+action GetList, UpdateList, DeleteList, CreateTask, UpdateTask, DeleteTask, EditShares { 
+    principal: [User], 
+    resource:[List]
```
Comment (cdisselkoen) [inline text/0024-schema-syntax.md:49]:
```suggestion
    resource: [List] 
```

File: text/0024-schema-syntax.md [hunk 2]
```diff
@@ -183,7 +191,7 @@ action ViewDocument in [ReadActions]
   attributes {
     version: 1,
     group: "meta"
-  };
+  }
```
Comment (cdisselkoen) [inline text/0024-schema-syntax.md:194]: why this? don't we still need the closing `;`?

**Review by cdisselkoen — APPROVED** [review]
Looks good

File: text/0024-schema-syntax.md (hunk 2 above)
Comment (khieta) [reply text/0024-schema-syntax.md:194]: Also, I think the braces are mismatched -- `attributes` should be at the same nest level as `principal` etc.

Comment (aaronjeline) [reply text/0024-schema-syntax.md:194]: No, I don't think it can be, otherwise how to specify an "empty" body.

Comment (cdisselkoen) [reply text/0024-schema-syntax.md]:
```
action View {
  attributes: {
    readOnly: true
  }
};
```
?  

ie it just omits `principal`, `resource`, and `context`

Comment (cdisselkoen) [reply text/0024-schema-syntax.md]: the alternative `action View {} { attributes: ... };` is also fine with me though

Comment (cdisselkoen) [reply text/0024-schema-syntax.md]: er I mean `action View {} attributes { ... };`

**Review by andrewmwells-amazon — APPROVED** [review]

**Comment by aaronjeline** [comment]
Decided against this small tweak.

**Pull request closed without merging**